src/applications/facility-locator/components/FacilityTypeDropdown.jsx

```diff
@@ -79,22 +113,48 @@ class FacilityTypeDropdown extends Component {
 
           return (
             <div>
-              <label
-                aria-live="polite"
-                aria-relevant="additions"
-                htmlFor="facility-dropdown-toggle"
-                id="facility-dropdown-label"
-              >
-                Search for
+              <div className="row">
+                <div className="columns medium-4">
+                  <label
+                    aria-live="polite"
+                    aria-relevant="additions"
```

Comment:
We will want to move these `aria-live` and `aria-relevant` attributes down to `<div id="infoBubble">` after having a chance to test with a screen reader. It's the containing element, and causes screen readers to read the entire popup.

The item in question is on line 164, approximately. I changed its markup to look like the following and verified with VoiceOver:

```html
<div
  aria-live="polite"
  aria-relevant="additions"
  id="infoBubble"
>
```